torch/onnx/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 producer_name = "pytorch"
 producer_version = _C._onnx.PRODUCER_VERSION
-constant_folding_opset_versions = [9, 10, 11, 12, 13, 14]
+constant_folding_opset_versions = [9, 10, 11, 12, 13, 14, 15]
```

Comment:
Could this list ever be contiguous? If not, lets ask python to generate it
```suggestion
constant_folding_opset_versions = list(range(9, 16))
```